R/as_polars.R

```diff
@@ -303,11 +303,12 @@ as_polars_lf.RPolarsLazyGroupBy = function(x, ...) {
 
 #' To polars Series
 #'
-#' [as_polars_series()] is a generic function that converts an R object to a
-#' polars Series. It is basically a wrapper for [pl$Series()][pl_Series].
-#' @param x Object to convert into a polars Series
-#' @param name A string to use as the name of the Series.
-#' If `NULL` (default), the name of `x` is used or an unnamed Series is created.
+#' [as_polars_series()] is a generic function that converts an R object to
+#' [a polars Series][Series_class].
```

Comment:
Should we have a list of R objects that can be converted to Series?

Reply:
Definitely needed... See #425.
The documentation for the `arrow` package is helpful.

Reply:
Ok, we can address #425 in full in another PR then